README.md

```diff
@@ -73,6 +73,8 @@ cd oodeel
 make prepare-dev
 ```
 
+> **Note:** If you want to install oodeel for a specific backend, replace the last line with either `make prepare-dev-torch` (for torch users) or `make prepare-dev-tf` (for tensorflow users).
+
```

Comment:
Ok for me. Just a suggestion, why don't we put "dev" stuff into CONTRIBUTING.md instead?

Reply:
Solved in #64